### PR TITLE
Register UID even if user is steam linked

### PR DESCRIPTION
**Pull request by Rackover**

### CODE REVIEW

File: server/lobbyconnection.py [hunk 1]
```diff
@@ -525,9 +525,9 @@ def check_version(self, message):
                     "player_id": player_id
                 })
 
-            if not self.player_service.is_uniqueid_exempt(player_id) and steamid is None:
```
Comment (Geosearchef) [inline server/lobbyconnection.py]: That won't work. `check_policy_conformity` sends an error to the user when the system is already registered, meaning steam bypass won't work anymore, this method will continue running, but the user gets an error anyways.

Comment (Rackover) [reply server/lobbyconnection.py]: Ok let's try this way

**Comment by Rackover** [comment]
Welp :p And now, to update tests..

**Comment by Askaholic** [comment]
Superseded by #510.

**Pull request closed without merging**